README.md

```diff
@@ -7,6 +7,18 @@
 
 `yarn` followed by `yarn build` should work on systems with a shell.
 
+## Run with docker
+
+Building the image will build the plugin and copy it into the grafana/plugins folder.
+
+`$ docker build -t cognite-grafana .`
+
+`$ docker run -d --name grafana -p 3000:3000 cognite-grafana`
+
+Now you can access it at http://localhost:3000
```

Comment:
Make this a link?

Reply:
Github will parse that as a link by default